src/app/utils/wait.ts

```diff
@@ -0,0 +1,5 @@
+export function wait(tShirtSize: number) {
+  return new Promise((resolve) => {
+    setTimeout(resolve, tShirtSize);
+  });
+}
```

Comment:
The parameter name ```tShirtSize``` is somewhat misleading in the context of a wait function.
```suggestion
export function wait(duration: number) {
  return new Promise((resolve) => {
    setTimeout(resolve, duration);
  });
}
```